src/components/Filter/Filter.scss

```diff
@@ -0,0 +1,23 @@
+@import '../../assets/css/colors';
+
+$color-green-dark: #265732;
+$color-gray-light: #dadada;
+
+.filterHeading {
+  background-color: $color-green-light !important;
+  color: $color-green-dark !important;
```

Comment:
Forslag. Begynn med å slette .filterHeading fra Filter.scss. Flytt fargene inn i _colors.scss. Lag følgende filer i src/assets/css:

src/assets/css/_panels.scss:
```
@import './colors';

.panel {
  &.panel-success {
    & > .panel-heading {
      background-color: $color-green-light;
      color: $color-green-dark;
    }
  }
}
```

src/assets/css/app.scss:
```
@import "panels";
```

Åpne opp .bootstraprc (i root folder), og finn linjen (som er utkommentert) med appStyles. Kommenter den inn og la den være som følger:
```
appStyles: ./src/assets/css/app.scss
```

Nå vil alle panel-headings i en panel-success ha disse fargene.

Selvfølgelig, dersom det ikke er meningen, så kan du jo bare beholde det slik det er.
Forøvrig, det hadde kanskje vært bedre å redefinere panel-default, hvis dette skal være default heading?
Kanskje vi kunne definere en for lærer og en for elev (altså ikke nødvendigvis overskrive, men lage nye?)

Uansett, du skjønner tegninga. Endre på eller legge til bootstrap sitt oppsett, istedetfor å bruke !important for å overstyre. Dermed trenger vi ikke overstyre overalt, men definere en standard via app.scss og alle imports derfra.